src/devices/bus/technics/hdae5000.cpp

```diff
@@ -0,0 +1,110 @@
+// license:BSD-3-Clause
+// copyright-holders:Antoine Mine, Olivier Galibert, Felipe Sanches
+//
+// HD-AE5000, Hard Disk & Audio Extension for Technics KN5000 emulation
+//
+// The HD-AE5000 was an extension board for the Technics KN5000 musical keyboard.
+// It provided a hard-disk, additional audio outputs and a serial port to interface
+// with a computer to transfer files to/from the hard-drive.
+
+#include "emu.h"
+#include "bus/ata/hdd.h"
+#include "hdae5000.h"
+#include "machine/i8255.h"
+
+namespace {
+
+class hdae5000_device : public device_t, public kn5000_extension_interface
```

Comment:
If I'm gathering right it should have `static constexpr feature_type unemulated_features() { return feature::DISK | feature::SOUND; }`

Reply:
thanks!